index.html

```diff
@@ -13,6 +13,8 @@
 <body>
   <h1>Cool Cats Coding!</h1>
   <p>OH BOY</p>
+  <h2>Alex McKenzie</h2>
```

Comment:
I think h2 makes sense here, since h1 should only appear once per page.

Reply:
LGTM